CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+[asdf-vm]: https://asdf-vm.com/
+
+# 🚀 Getting Started
+
+These instructions will get you a copy of the project up and running on your
+local machine for development and testing purposes.
+
+## 📥 Prerequisites
+
+The following software is required to be installed on your system:
+
+- [Next.js](https://nextjs.org/) (using [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app))
+
+- [React-Big-Calendar](https://github.com/jquense/react-big-calendar)
+
+- [Ant Desgin](https://ant.design/docs/react/introduce) ([Documentation](https://github.com/ant-design/ant-design/))
+
+- [EmailJs](https://www.emailjs.com/)
+
+- [html2pdf](https://github.com/eKoopmans/html2pdf.js)
+
+- [jsPDF](https://github.com/parallax/jsPDF)
```

Comment:
Here should only be Node.js, since it's the only external dependency. You can see how I made it [here](https://github.com/cesium/codeweek/blob/main/CONTRIBUTING.md).

Reply:
Done!